3.6/oasis/backups.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+description: Quick guide about backups in Oasis for ArangoDB.
+title: Backups in Oasis
+---
+
+# How to create backups
+
+To backup data in Oasis for an ArangoDB installation, navigate to the backup section of your deployment created previously.
+
+![Backup ArangoDB](../images/oasis-backup-section.png)
+
+There are two types of backups. Periodic backups taken through backup policies and manual backups taken by hand.
+
+## Periodic backups
+
+Periodic backups are taken on a given timeframe via a Backup Policy. To see when the new backup is due, observe the schedule section.
+
+![Backup Policy schedule](../images/oasis-backup-policy-schedule.png)
+
+Upon creating a deployment, by default, a new policy is created which will take backups of a deployment hourly every two hours. To edit this policy (or any policy), highlight it in the row above and hit the pencil icon.
```

Comment:
When a new deployment is created, a default backup policy is created for it as well.
This policy will create backups every two hours.....

Reply:
I thought that might be weird phrasing. It was taken from this:
```
Hourly every 2 hour(s)
```

Which is displayed in the schedule.